packages/cysync-core/tsconfig.eslint.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
     "outDir": "dist",
-    "lib": ["ES2015"]
+    "lib": ["ES2015"],
+    "paths": {
+      "~/*": ["src/*"]
+    }
```

Comment:
This change is not necessary since we are already extending `tsconfig.json` which has this change.